lesson0/task4/src/main/java/EvenOrOdd/EvenOrOdd.java

```diff
@@ -2,19 +2,19 @@
 
 public class EvenOrOdd {
     public static void main(String[] args) {
-        System.out.println(evenOdd(5));
+        System.out.println(isEven(5));
     }
 
-    public static String evenOdd(int numberInput) {
-        String feedback = "";
+    public static boolean isEven(int numberInput) {
```

Comment:
![image](https://user-images.githubusercontent.com/17989854/119775275-b042cc80-bec3-11eb-9452-cf2983114576.png)

To jest jak najbardzie ok, ale można by pokusić się tutaj o skrócenie tego do:

`return numberInput % 2 == 0`